### ISSUE TITLE
Pluggin installation failure

### ISSUE THREAD

**Issue by lineldcosta** [jpuri/react-draft-wysiwyg#385]
draftjs-utils" doesn't exist issue is coming while installing the package

**Comment by jpuri** [comment]
@lineldcosta 

Which version of editor are you using, also can you plz paste stack trace

**Comment by lineldcosta** [comment]
lionel@lionel-Lenovo-G580:/var/www/reloaded/reloaded$ yarn add react-draft-wysiwyg
yarn add v0.21.3
[1/4] Resolving packages...
error "/var/www/reloaded/draftjs-utils" doesn't exist

**Comment by mhurwi** [comment]
Hi, trying to experiment with this package and I'm getting the same issue, cannot install 'draft-utils'.  

Here's the npm debug:

```
0 info it worked if it ends with ok
1 verbose cli [ '/Users/mhurwi/.nvm/v8.1.3/bin/node',
1 verbose cli   '/Users/mhurwi/.nvm/v8.1.3/bin/npm',
1 verbose cli   'install',
1 verbose cli   '-S',
1 verbose cli   'react-draft-wysiwyg' ]
2 info using npm@5.0.3
3 info using node@v8.1.3
4 verbose npm-session 63cdfe7b7c212eb2
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 http fetch GET 200 https://registry.npmjs.org/react-draft-wysiwyg 321ms
8 silly pacote tag manifest for react-draft-wysiwyg@latest fetched in 359ms
9 silly install loadIdealTree
10 silly install cloneCurrentTreeToIdealTree
11 silly install loadShrinkwrap
12 silly install loadAllDepsIntoIdealTree
13 silly resolveWithNewModule react-draft-wysiwyg@1.10.3 checking installable status
14 silly fetchPackageMetaData error for draftjs-utils@file:../draftjs-utils Could not install from "node_modules/draftjs-utils" as it does not contain a package.json file.
15 verbose stack Error: ENOENT: no such file or directory, open '/Users/mhurwi/Documents/code/ducktypecoder/app/node_modules/draftjs-utils/package.json'
16 verbose cwd /Users/mhurwi/Documents/code/ducktypecoder/app
17 verbose Darwin 16.6.0
18 verbose argv "/Users/mhurwi/.nvm/v8.1.3/bin/node" "/Users/mhurwi/.nvm/v8.1.3/bin/npm" "install" "-S" "react-draft-wysiwyg"
19 verbose node v8.1.3
20 verbose npm  v5.0.3
21 error code ENOLOCAL
22 error Could not install from "node_modules/draftjs-utils" as it does not contain a package.json file.
23 verbose exit [ 1, true ]
```

**Comment by ZeroCho** [comment]
I deleted `node_modules` and `yarn.lock`
and tried `yarn install`

yarn install v0.27.5
info No lockfile found.
[1/4] Resolving packages...
error "C:\\Users\\zerocho\\WebstormProjects\\draftjs-utils" doesn't exist.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

**Comment by lqm229026** [comment]
@jpuri same issue
npm: v5.1.0
cmd: npm install --save react-draft-wysiwyg
os: win7
stack:
```
npm ERR! code ENOLOCAL
npm ERR! Could not install from "node_modules\draftjs-utils" as it does not cont
ain a package.json file.
```

**Comment by orangeyyy** [comment]
I think the the owner fix this bug in 1.10.4, but he seem to forget publish it on npm.

**Comment by jpuri** [comment]
Thanks @ lot @orangeyyy , fix is on npm now 😄 

Sorry for the inconvenience everyone.
